ui/lib/kv/addon/components/page/secret/metadata/version-history.hbs

```diff
@@ -5,7 +5,12 @@
     <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
   </:tabLinks>
 </KvPageHeader>
+
+<Toolbar />
```

Comment:
Seems sensible to add this for styling purposes since the other tabs have a toolbar